overlord/assertstate/assertmgr.go

```diff
@@ -45,8 +49,7 @@ type AssertManager struct {
 func Manager(s *state.State) (*AssertManager, error) {
 	runner := state.NewTaskRunner(s)
 
-	runner.AddHandler("fetch-snap-assertions", doFetchSnapAssertions, undoFetchSnapAssertions)
-	// TODO: check-snap-assertions handlers
+	runner.AddHandler("fetch-check-snap-assertions", doFetchCheckSnapAssertions, undoFetchCheckSnapAssertions)
```

Comment:
Can we use `"validate-snap"` here?  It's shorter, seems a bit more inline with the remaining snap tasks, it's quite "legally binding", and it takes a small step back from the nitty-gritty of the operation, meaning some freedom to operate internally.